test/artifacts.jl

```diff
@@ -316,12 +316,12 @@ end
     # the global artifact namespace and package list, but it should be harmless.
     mktempdir() do project_path
         with_pkg_env(project_path) do
-            copy_test_package(project_path, "ArtifactInstallation")
+            path = git_init_package(project_path, joinpath(@__DIR__, "test_packages", "ArtifactInstallation"))
             add_this_pkg()
             Pkg.add(Pkg.Types.PackageSpec(
                 name="ArtifactInstallation",
                 uuid=Base.UUID("02111abe-2050-1119-117e-b30112b5bdc4"),
-                path=joinpath(project_path, "ArtifactInstallation"),
+                path=path,
```

Comment:
What were the errors you were seeing?  I'm not sure why `git_init_package()` would have any different behavior here from `copy_test_package()`?

Reply:
Thanks for reviewing. `Pkg.add` requires the package to be git version controlled because it always clones (it doesn't track the path directly like `Pkg.develop` does).